src/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -154,7 +154,7 @@ public static SafeFreeCredentials AcquireCredentialsHandle(X509Certificate certi
 
                 // CoreFX: always opt-in SCH_USE_STRONG_CRYPTO except for SSL3.
                 if (((protocolFlags == 0) ||
-                        (protocolFlags & (Interop.SChannel.SP_PROT_TLS1_0 | Interop.SChannel.SP_PROT_TLS1_1 | Interop.SChannel.SP_PROT_TLS1_2)) != 0)
+                        (protocolFlags & (Interop.SChannel.SP_PROT_TLS1_0 | Interop.SChannel.SP_PROT_TLS1_1 | Interop.SChannel.SP_PROT_TLS1_2 | Interop.SChannel.SP_PROT_TLS1_3)) != 0)
```

Comment:
Is the comment correct?  And if so, to prevent this sort of missing value in the future, should we invert the condition to be more like:
```C#
if ((protocolFlags & Interop.SChannel.SP_PROT_SSL3) == 0)
```
?

Reply:
@stephentoub - makes sense to write it this way - I'm not sure if SP_PROT_SSL2 should not also be included since we aren't checking for that, would be nice if someone with more context commented here

Reply:
@davidsh, @bartonjs, any more insight on this?

Reply:
I'm always in favor of more extensible.  In this case I think it would be

```C#
if (protocolFlags == 0 || (protocolFlags & ~(SSL2 | SSL3)) != 0)
```

Since the test is actually is "any of the TLS flags are set", vs "SSL2/SSL3 isn't set".  (protocolFlags = SSL3 | TLS1) looks to be `true`.